common/src/main/java/org/opensearch/ml/common/MLTaskState.java

```diff
@@ -30,5 +30,6 @@ public enum MLTaskState {
     CANCELLED,
     COMPLETED_WITH_ERROR,
     CANCELLING,
-    EXPIRED
+    EXPIRED,
+    UNREACHABLE
```

Comment:
Can you check if some code is missing in this PR to support this new state? This is a new state only added in this PR, but I don't see any "UNREACHABLE" assigned in this PR. Are you missing some commits from your local branch?

Reply:
It is being used here https://github.com/opensearch-project/ml-commons/pull/3421/files#diff-2b93fc38b2ea30a5cd7ebfd0731373649c15f7f3cf60b09c978d4afaeceb5e17R485.